runtime/src/app/app.ts

```diff
@@ -364,8 +373,8 @@ function load_css(chunk: string) {
 	});
 }
 
-export function load_component(component: ComponentLoader): Promise<{
-	default: ComponentConstructor,
+export function load_component(component: DOMComponentLoader): Promise<{
```

Comment:
This seems like it's actually returning `Promise<DOMComponentModule>`, right?

Can do this in a follow up

Reply:
updated